lib/genome/importers/hingorani_casas/hingorani_casas_tsv_importer.rb

```diff
@@ -0,0 +1,26 @@
+module Genome
+  module Importers
+    module HingoraniCasas
+      def self.source_info
+        {
+          base_url:           'http://www.ncbi.nlm.nih.gov/gene?term=',
+          site_url:           'http://stm.sciencemag.org/content/9/383/eaag1166',
+          citation:           'The druggable genome and support for target identification and validation in drug development. Finan C, Gaulton A, Kurger F, Lumbers T, Shah T, Engmann J, Galver L, Kelly R, Karlsson A, Santos R, Overington J, Hingorani A, Casas JP.   Sci. Transl. Med. 2017 Mar 29;9(383):eaag1166 PMID: 28356508.',
+          source_db_version:  '31-May-2017',
+          source_type_id:     DataModel::SourceType.POTENTIALLY_DRUGGABLE,
+          source_db_name:     'HingoraniCasas',
+          full_name:          'The druggable genome and support for target identification and validation in drug development (Hingorani & Cass, 2017)'
+        }
+      end
+
+      def self.run(tsv_path)
+        TSVImporter.import tsv_path, HingoraniCasasRow, source_info do
+          gene :gene_symbol, nomenclature: 'HingoraniCasas Gene Symbol' do
+            name :gene_symbol, nomenclature: 'Gene Symbol', transform: ->(x) { x.upcase }
+            name :ensembl_id, nomenclature: 'Ensembl Id', transform: ->(x) { x.upcase }
+          end
```

Comment:
@kcotto Its a little clunky, but you need to make a call to `save!` on the closing `end` of this block. That will cause the importer to actually commit to the database. You can see an example of this [here](https://github.com/griffithlab/dgi-db/blob/master/lib/genome/importers/chembl/chembl_tsv_importer.rb#L45) in another importer.

Adding that should fix the issue you're seeing!

On GitHub, any additional commits you push to this branch will simply be appended to this PR so you can make the fix and push it as another commit to this branch and that will work just fine! Otherwise, on a PR where you're the only one working on this branch, you could `git amend` your existing fix and force push to this branch (though you definitely don't want to do that on `master` or when you're collaborating on a branch as it will overwrite history.) 

Let me know if you run into any issues with that!